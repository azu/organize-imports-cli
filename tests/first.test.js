import { test, expect } from '@jest/globals';
import runCli from "./run-cli.js";

test("The first Test - processes file correctly", async () => {
  const result = await runCli("cli/first", ["file1.js"]);
  expect(result.status).toMatchSnapshot();
  expect(result.stdout).toMatchSnapshot();
  expect(result.stderr).toMatchSnapshot();
  expect(result.write).toMatchSnapshot();
});
