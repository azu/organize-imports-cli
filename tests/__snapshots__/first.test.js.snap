// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The first Test - processes file correctly 1`] = `1`;

exports[`The first Test - processes file correctly 2`] = `""`;

exports[`The first Test - processes file correctly 3`] = `"appendStdout is not defined"`;

exports[`The first Test - processes file correctly 4`] = `[]`;
